### PR TITLE
Tweaked and added image grid component

### PR DESCRIPTION
**Pull request by TaylorHorn**
screen shot 2017-09-19 at 10 12 05 am

### CODE REVIEW

File: src/tags/rtb-image-grid/index.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
+<html>
+
+  <head>
+
+    <title>Image Grid</title>
+
+    <style>
+
+      body {
+        font-family: 'Work Sans', sans-serif;
```
Comment (MattStopa) [inline src/tags/rtb-image-grid/index.html]: You can't set the the font family int he tag itself.

File: src/tags/rtb-image-grid/rtb-info-box.tag [hunk 2]
```diff
@@ -0,0 +1,87 @@
+<rtb-info-box>
+
+  <style>
+
+    .mydiv0 {
```
Comment (MattStopa) [inline src/tags/rtb-image-grid/rtb-info-box.tag]: mydiv is a bad name. Needs to be more generic.

File: src/tags/rtb-image-grid/rtb-info-box.tag [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+      </div>
+    </div>
+
+    <div class="unit whole no-gutters">
+      <h3>INDUSTRIES</h3>
+      <p>{opts.data.industries}</p>
+    </div>
+  </div>
+
+  <script>
```
Comment (MattStopa) [inline src/tags/rtb-image-grid/rtb-info-box.tag]: random empty script tag

File: src/tags/rtb-image-grid/rtb-info-box.tag [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+      </div>
+    </div>
+
+    <div class="grid">
+      <div class="unit one-fifth no-gutters">
+        <h3>PlATFORMS</h3>
+        <p>{opts.data.platforms}</p>
+      </div>
+      <div class="unit one-fifth">
+        <h3>TECH HIGHLIGHTS</h3>
```
Comment (MattStopa) [inline src/tags/rtb-image-grid/rtb-info-box.tag]: Why is this stuff hard coded?

**Review by MattStopa — CHANGES_REQUESTED** [review]

**Comment by TaylorHorn** [comment]
screen shot 2017-09-21 at 11 40 41 am

**Pull request merged**